docs/source/pvtdatastorage.rst

```diff
@@ -0,0 +1,16 @@
+Private Data Storage Enhancements
+=================================
+
+Private Data Storage was also enhanced with CouchDB
+
+Enhancement 1
```

Comment:
can we name it better than just saying "Enhancement 1" ?

Reply:
done